Modules/_testcapi/immortal.c

```diff
@@ -26,7 +26,17 @@ static PyObject *
 test_immortal_small_ints(PyObject *self, PyObject *Py_UNUSED(ignored))
 {
     for (int i = -5; i <= 256; i++) {
-        assert(verify_immortality(PyLong_FromLong(i)));
+        PyObject *obj = PyLong_FromLong(i);
+        assert(verify_immortality(obj));
+        int has_int_immortal_bit = ((PyLongObject *)obj)->long_value.lv_tag & (1 << 2);
```

Comment:
```suggestion
        int has_int_immortal_bit = ((PyLongObject *)obj)->long_value.lv_tag & IMMORTALITY_BIT_MASK;
```

Reply:
Done. This did require to include the internal header file `internal/pycore_long.h` in the tests. There are a few other capi tests that include internal headers, so I think this is ok.